docs/install/pxe-boot-install.md

```diff
@@ -179,9 +179,25 @@ boot
 
 This assumes the iPXE script is stored in `/usr/share/nginx/html/harvester/ipxe-join`.
 
-
 ## DHCP Server Configuration
 
+:::note
+
+In the PXE installation scenario, the `option routers` of DHCP server in following configuration is a **mandatory option**, which is used to install `default route` in Harvester host OS. The node will fail to start without it.
+
+In the ISO installation scenario, when the management network interface is in DHCP mode, the `option routers` is also a **mandatory option**.
+
+For example:
```

Comment:
```suggestion
You are required to use the *routers* option (`option routers`) when configuring the DHCP server. This mandatory option, which specifies a list of IP addresses for routers in order of preference, is used to configure the default route on the Harvester host. Without the default route, the node will fail to start.

Example:
```

Reply:
- The text should not be in a note.
- Mentioning ISO installation in this topic seems unnecessary.

Reply:
In ISO mode, interface can select `dhcp` or `static`, but in PXE mode, it can only use `dhcp`. Hence we describe both of them separately.

Reply:
Agree. We should provide information that is relevant to ISO installation in *iso-install.md*, and information that is relevant to PXE in *pxe-boot-install.md*. That's why I said that mentioning ISO installation here seems unnecessary.

Reply:
There is another consideration:

The `iso-install.md`  https://docs.harvesterhci.io/v1.1/install/iso-install#installation-steps  is difficult to link to detailed step (e.g. step 6) directly, when say `step 6` of the `installation-steps`, it is a bit cumbersome and further change of installation steps may break it silently.

Put the information here, is for user convenience to link from other places straightforwardly. And they are all related the `dhcp server configuration`, used in both `ios mode` and `pxe mode`.